packages/renderer/src/lib/container/ContainerList.spec.ts

```diff
@@ -846,11 +846,6 @@ test('Sort containers based on selected parameter', async () => {
   await fireEvent.click(image);
 
   expect(container2.compareDocumentPosition(container1)).toBe(2);
-
```

Comment:
This removal seems suspect to me

Reply:
@jeffmaury this is the part that probably have caused issues with others' builds like @deboer-tim mentioned, so I'm removing it to see if it helps

Reply:
OK accepting because all others PRs are failing but we should understand the cause and create a follow up PR